src/main/java/team/chisel/common/util/GenerationHandler.java

```diff
@@ -0,0 +1,61 @@
+package team.chisel.common.util;
+
+import java.util.Random;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraft.world.chunk.IChunkGenerator;
+import net.minecraft.world.chunk.IChunkProvider;
+import net.minecraft.world.gen.feature.WorldGenMinable;
+import net.minecraftforge.fml.common.IWorldGenerator;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
+import team.chisel.common.config.Configurations;
+
+public class GenerationHandler implements IWorldGenerator {
+
+	@Override
+	public void generate(Random random, int chunkX, int chunkZ, World world, IChunkGenerator chunkGenerator, IChunkProvider chunkProvider) {
+		switch (world.provider.getDimension()) {
+		case -1:
+			break;
+		case 0:
+			generateSurface(world, random, chunkX * 16, chunkZ * 16);
+			break;
+		case 1:
+			break;
+		}
```

Comment:
I'd like to be able to configure generation in other worlds, too...

Like Basalt in the nether.